crates/serpen/tests/fixtures/ast_rewriting/variable_name_collision/main.py

```
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+"""
+Variable name collision test.
+Both modules define variables with the same names.
+"""
+
+from constants import API_URL, VERSION, DEFAULT_TIMEOUT
+from config import API_URL as CONFIG_API_URL, VERSION as CONFIG_VERSION, MAX_RETRIES
+
+
+def main():
```

Ruff (0.11.9)

Missing return type annotation for public function `main`

(ANN201)